content/en/blog/_posts/2021-04-08-introducing-indexed-jobs.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Introducing Indexed Jobs"
+date: 2021-05-05
+slug: introducing-indexed-jobs
+---
+
+**Author:** Aldo Culquicondor (Google)
+
+Once you have containerized a non-parallel {{< glossary_tooltip text="Job" term_id="job" >}},
+it is quite easy to get it up and running on Kubernetes without modifications to
+the binary. But when running parallel distributed Jobs, in most cases, you had
+to set a separate system to partition the work among the workers. For
+example, you could set up a task queue to [assign one work item to each
+Pod](/docs/tasks/job/coarse-parallel-processing-work-queue/) or [multiple items
+to each Pod until the queue is emptied](/docs/tasks/job/fine-parallel-processing-work-queue/).
+
+SIG Apps introduces the Job _completion mode_, which allows you to control how
+Pod completions affect the overall progress of a Job and, with 2 possible values
+(for now):
+
+- `NonIndexed` (default): the Job is considered complete when there have been
+  a number of successfully completed Pods equal to the specified number in
+  `.spec.completions`. In other words, each Pod completion is homologous to
+  each other. Any Job you might have created before the introduction of
+  completion modes is implicitly NonIndexed.
+- `Indexed`: the Job is considered complete when there is one successfully
+  completed Pod associated to each index from 0 to `.spec.completions-1`. Each
+  Pod gets its associated index in the annotation
+  `batch.kubernetes.io/job-completion-index`.
```

Comment:
The index is exposed in `batch.kubernetes.io/job-completion-index` annotation.

Reply:
added "to each Pod"